src/nodes/agent/handlers/VaultsGitInfoGet.ts

```diff
@@ -28,7 +28,7 @@ class VaultsGitInfoGet extends RawHandler<{
     _cancel,
     meta: Record<string, JSONValue> | undefined,
     _ctx: ContextTimed, // TODO: use
```

Comment:
This has a `TODO: use`, what's the situation here? @tegefaulkes @amydevs

Reply:
i have no idea

Reply:
It should use the CTX for timing out the handler.

Reply:
ah alright, this should probably be added

Reply:
this will have to be added later in another PR